booking/src/main/java/org/dcsa/conformance/standards/booking/action/StateChangingBookingAction.java

```diff
@@ -3,12 +3,19 @@
 import org.dcsa.conformance.core.traffic.ConformanceExchange;
 import org.dcsa.conformance.standards.booking.checks.ScenarioType;
 
+import java.util.Set;
+
 public abstract class StateChangingBookingAction extends BookingAction {
   public StateChangingBookingAction(String sourcePartyName, String targetPartyName, BookingAction previousAction,
                                     String actionTitle, int expectedStatus) {
     super(sourcePartyName, targetPartyName, previousAction, actionTitle, expectedStatus);
   }
 
+  public StateChangingBookingAction(String sourcePartyName, String targetPartyName, BookingAction previousAction,
+                                    String actionTitle, Set<Integer> expectedStatus) {
+    super(sourcePartyName, targetPartyName, previousAction, actionTitle, expectedStatus);
```

Comment:
Same here. Please adjust the callers of the constructor to only use the changed Set<Integer> expectedStatus.
If you can use `List` instead, please also adopt.

Reply:
Yes, if most cases indeed are using one parameter on `expectedStatus`, than it makes sense. 
Using a Set makes sense if it we actively need to prevent duplicated values. I am fine with it, if it might happen.